load_results_into_pky_projects.R

```diff
@@ -0,0 +1,279 @@
+library(tidyverse)
+library(lubridate)
+library(dotenv)
+library(REDCapR)
+library(openxlsx)
+library(sendmailR)
+
+source("functions.R")
+
+# echo details from the .env file we read
+Sys.getenv("INSTANCE")
+Sys.getenv("PROJECT")
+
+# set the timezone
+Sys.setenv(TZ = Sys.getenv("TIME_ZONE"))
+
+script_run_time <- strftime(Sys.time(), format = "%Y%m%d_%H%M") 
+
+# email credentials
+email_server <- list(smtpServer = Sys.getenv("SMTP_SERVER"))
+email_from <- Sys.getenv("EMAIL_FROM")
+email_to <- unlist(strsplit(Sys.getenv("EMAIL_TO")," "))
+email_cc <- unlist(strsplit(Sys.getenv("EMAIL_CC")," "))
+email_subject <- paste(Sys.getenv("EMAIL_SUBJECT"), script_run_time)
+
+# read data from survey project
+survey_project_read <- redcap_read_oneshot(redcap_uri = 'https://redcap.ctsi.ufl.edu/redcap/api/',
+                               token = Sys.getenv("SURVEY_TOKEN"))$data %>%
+  filter(!is.na(research_encounter_id)) %>%
+  mutate(test_date_and_time=ymd_hms(test_date_and_time)) %>%
+  select(record_id,
+         redcap_event_name,
+         ce_firstname,
+         ce_lastname,
+         icf_age,
+         patient_dob,
+         icf_lar_name,
+         icf_lar_relationship,
+         ce_orgconsentdate,
+         icf_email,
+         qpk_phone,
+         research_encounter_id,
+         test_date_and_time,
+         covid_19_swab_result)
+
+# If data already in Survey got flushed from Serial (generally due to testing), load it into serial now
+old_records_for_serial <- survey_project_read %>%
+  filter(redcap_event_name == "baseline_arm_1") %>%
```

Comment:
Are we only concerned with baseline records for serial project. What if a subject needs a retest? @pbchase

Reply:
`old_records_for_serial` is most valuable in _testing_. It might never do anything useful in a production workflow, but it is also harmless in a production workflow.

We seed the baseline swab-positive records into serial. After that, serial generates it's own research_encounter_ids that are matched in `swab_result_for_serial_spawned_from_serial`